### DISCUSSION TITLE
Can't see / start frontend

### DISCUSSION THREAD

**Discussion by vongomben** [Koenkk/zigbee2mqtt#16182] [General]
Hi. 
I'm on a pi3, inwhich I installed zigbee2mqtt alongside magicmirror and node-red

I'm able to use zigbee2mqtt wothout frontend, but when I try to add it, nothing happens. 

if I add  `frontend: true` all the flow keeps on working, but sometimes I receive a `{"state":"offline"}`.

If I add in the ` /opt/zigbee2mqtt/data/configuration.yaml` 

```frontend: true

frontend:
	port: 8888
	host: 0.0.0.0
```

Everything stops working. 

The ports I listen on this machine are not influenced anyway, alre always these. 

```
sudo lsof -i -P -n | grep LISTEN

cupsd      594  root    6u  IPv6  11955      0t0  TCP [::1]:631 (LISTEN)
cupsd      594  root    7u  IPv4  11956      0t0  TCP 127.0.0.1:631 (LISTEN)
vncserver  618  root   11u  IPv6  11982      0t0  TCP *:5900 (LISTEN)
vncserver  618  root   12u  IPv4  11983      0t0  TCP *:5900 (LISTEN)
sshd       633  root    3u  IPv4  14478      0t0  TCP *:22 (LISTEN)
sshd       633  root    4u  IPv6  14480      0t0  TCP *:22 (LISTEN)
node-red   705   pi3   19u  IPv6  18056      0t0  TCP *:1883 (LISTEN)
node-red   705   pi3   20u  IPv4  18047      0t0  TCP *:1880 (LISTEN)
electron   984   pi3   68u  IPv4  16919      0t0  TCP 127.0.0.1:8080 (LISTEN)

```

Any help?

**Comment by EvertDekker** [comment]
Stupid answer of the day:
Shouldn't the indentation for the port and host line be 2 spaces?
